api/src/main/java/com/epam/pipeline/manager/docker/DockerRegistryManager.java

```diff
@@ -490,7 +490,7 @@ private DockerRegistry loadByIdOrName(DockerRegistry registry) {
                     toolGroup.getName()));
             toolManager.updateToolVersionScanStatus(toolInGroup.get().getId(),
                     ToolScanStatus.NOT_SCANNED, DateUtils.now(), event.getTarget().getTag(),
-                    null, event.getTarget().getDigest());
+                    null, null, event.getTarget().getDigest());
```

Comment:
I see a lot of changes where `null` is added as argument to method `updateToolVersionScanStatus`. Maybe we should introduce overloaded method without `osVersion` argument  to reduce number of changes to the code?